Zend/tests/traits/constant_016.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Compatibility of values of same name trait constants is checked after their constant expressions are evaluated
+--FILE--
+<?php
+
+define('CONSTANT', 2);
+
```

Comment:
Have a similar test with a runtime defined constant to see if this borks?

Reply:
Actually, I chose `define()` with the intention of defining const at runtime here, but after the optimization of opcache, it seems to compile to the same code as const.
I will use eval to ensure it is defined at runtime.

Reply:
You could use something from the $_SERVER array

Reply:
Thanks for pointing this out! I've changed to use `$_ENV`